.docker/Makefile

```diff
@@ -6,10 +6,10 @@ TAG ?= latest
 all: judge-tier1 judge-tier2 judge-tier3
 
 judge-tier1:
-	cd tier1 && docker build --build-arg TAG="${GIT_TAG}" -t dmoj/judge-tier1 -t dmoj/judge-tier1:$(TAG) .
+	cd tier1 && docker build --build-arg TAG="${GIT_TAG}" -t vnoj/judge-tier1 -t vnoj/judge-tier1:$(TAG) .
```

Comment:
xD I think we should keep this the same name

Reply:
We made several important changes to our fork so I think we should change the name to avoid any mistakes later.

Reply:
make sense.